azure-client-runtime/src/main/java/com/microsoft/azure/v2/OperationStatus.java

```diff
@@ -6,54 +6,134 @@
 
 package com.microsoft.azure.v2;
 
+import com.microsoft.rest.protocol.SerializerAdapter;
+import com.microsoft.rest.v2.http.HttpRequest;
+import com.microsoft.rest.v2.http.HttpResponse;
+import rx.Single;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
 /**
- * The status of a long running operation. This generally is created from the result of polling for
- * whether a long running operation is done or not.
- * @param <T>
+ * The current state of polling for the result of a long running operation.
  */
-public final class OperationStatus<T> {
-    private final boolean isDone;
-    private final T result;
+class OperationStatus<T> {
```

Comment:
Is this no longer something you want referenced in Azure gen code?

Reply:
We definitely want this referenced in gen code. Fixed.